Ponleou

ponleou opened the pull request:
# task: defined system design to tailwind's configuration file
![Screen Shot 2023-09-20 at 10 39 31 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/5da11c42-4058-4ee6-a18c-f22b93e7364e)
![Screen Shot 2023-09-20 at 10 21 39 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/134fc6f7-adee-4222-b56b-16ad731a58e3)

# task: added icon support with Iconify
![Screen Shot 2023-09-22 at 8 14 53 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/84990ff3-7a8c-4ce5-bbd6-bf5de47f92fe)

# task: defined the logo component
![Screen Shot 2023-09-22 at 8 17 27 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/49b7cf86-9d83-4767-8958-0424191a69a8)

# task: added home and about pages
![Screen Shot 2023-09-20 at 10 29 44 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/ec2a84f8-c84b-42dc-8900-958ccb7af353)

# task: created navigation bar components

### Menu Item
![Screen Shot 2023-09-22 at 8 21 38 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/a61fd985-50a1-4d49-b05e-1db67ca670a8)
### Menu and Submenu Items
![Screen Shot 2023-09-22 at 8 22 36 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/4fbef51e-4479-4d33-9cc4-94a7a355ad4a)


# task: created the navigation bar
![Screen Shot 2023-09-22 at 8 19 20 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/650063ec-67fc-4cb3-8ac8-c81e6aa80bb3)
![Screen Shot 2023-09-22 at 8 19 49 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/9e6faa55-6a21-4ad7-8023-a177c141a716)
![Screen Shot 2023-09-22 at 8 20 12 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/e418483a-9811-43d4-8566-6860b97db486)

sovouthea33 commented on tailwind.config.js:
```diff
@@ -13,8 +13,32 @@ export default {
             fontFamily: {
-                sans: ['Figtree', ...defaultTheme.fontFamily.sans],
+                sans: ["Catamaran" ,'Figtree', ...defaultTheme.fontFamily.sans],
             },
```
add

masterskynetx10 commented on resources/css/app.css:
```diff
@@ -1,3 +1,8 @@
 @tailwind components;
 @tailwind utilities;
+
+@font-face {
```
@sovouthea33 on vecert we commit this file right?

masterskynetx10 commented:
@ponleou you're confusing don't screenshot your code. Screenshot the out come and what it looks like or what it function work

ponleou commented:
> @ponleou you're confusing don't screenshot your code. Screenshot the UI out come and what it looks like or what it function work

Hi bong, I updated the comments:

# task: added icon support with Iconify
![Screen Shot 2023-09-22 at 8 14 53 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/84990ff3-7a8c-4ce5-bbd6-bf5de47f92fe)

# task: defined the logo component
![Screen Shot 2023-09-22 at 8 17 27 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/49b7cf86-9d83-4767-8958-0424191a69a8)

# task: created navigation bar components

### Menu Item
![Screen Shot 2023-09-22 at 8 21 38 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/a61fd985-50a1-4d49-b05e-1db67ca670a8)
### Menu and Submenu Items
![Screen Shot 2023-09-22 at 8 22 36 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/4fbef51e-4479-4d33-9cc4-94a7a355ad4a)


# task: created the navigation bar
![Screen Shot 2023-09-22 at 8 19 20 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/650063ec-67fc-4cb3-8ac8-c81e6aa80bb3)
![Screen Shot 2023-09-22 at 8 19 49 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/9e6faa55-6a21-4ad7-8023-a177c141a716)
![Screen Shot 2023-09-22 at 8 20 12 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/e418483a-9811-43d4-8566-6860b97db486)

ponleou commented:
# task: added footer support to MenuItem and SubMenuItem components

### footerMenu style in MenuItem component
![Screen Shot 2023-09-22 at 11 02 14 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/f0cef4e1-6fe2-4622-b8cc-c923e5cd3821)

### footerSubmenu style in SubMenuItem component
![Screen Shot 2023-09-22 at 11 03 11 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/2a691165-5ebd-4974-8db6-ecb70a77f1cc)

# fix: added the block style to the ACMCertLogo for it to work with footer
![Screen Shot 2023-09-22 at 11 06 50 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/037fa86c-8371-4748-8b6a-11aff1caab4b)

# task: created the footer component
![Screen Shot 2023-09-22 at 11 09 20 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/235bb976-e2bc-4fea-89ac-f3133dc43c8e)
![Screen Shot 2023-09-22 at 11 09 24 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/b7619e70-a7f9-4283-aa6b-8032b0993e49)
![Screen Shot 2023-09-22 at 11 09 38 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/d5ad1669-c874-47ad-8620-547dc2d9d116)
![Screen Shot 2023-09-22 at 11 09 58 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/abfa6056-37f0-402c-a273-81d0d9232a9c)

masterskynetx10 commented:
@ponleou your PR title should be follow like your ticket task.

masterskynetx10 commented on resources/js/Components/navigation/Footer.vue:
```diff
@@ -0,0 +1,95 @@
+        <footer class="bg-white">
+            <div class="mx-auto w-full max-w-screen-xl p-8">
+                <ACMCertLogo class="sm:hidden"></ACMCertLogo>
+                <ul class="grid grid-cols-2 gap-8 py-6 sm:py-0 sm:flex sm:justify-between">
+                    <ul>
```
<ul> semantic is define list
```suggestion
                <div class="grid grid-cols-2 gap-8 py-6 sm:py-0 sm:flex sm:justify-between">
                    <ul>
```

masterskynetx10 commented on resources/js/Components/navigation/Footer.vue:
```diff
@@ -0,0 +1,95 @@
+                            <ACMCertLogo></ACMCertLogo>
+                        </li>
+                    </ul>
+                    <ul class="hidden sm:block">
```
```suggestion
                    <div class="hidden sm:block">
```

ponleou replied:
Hi bong, the MenuItem component is a list element, so when I change the ul to div, the list style is applied

![Screen Shot 2023-09-25 at 9 53 17 AM](https://github.com/ponleou/public-site-acmcert/assets/89851049/205fb454-46c9-4536-a0c1-90f0a404c62f)

Should I keep as ul or change to div and fix the list style?

ponleou replied:
Never mind, the fix was small.

masterskynetx10 commented on resources/js/Components/navigation/Footer.vue:
```diff
@@ -0,0 +1,95 @@
+                        </ul>
+                    </ul>
+                    <div class="sm:hidden">
+                        <SubMenuItem uniqueID=services-footer  name="Services" isFooterSubmenu="true">
```
when you pass variable please use like html attribute and value
```suggestion
                        <SubMenuItem uniqueID="services-footer"  name="Services" isFooterSubmenu="true">
```

masterskynetx10 commented on resources/js/Pages/About.vue:
```diff
@@ -0,0 +1,15 @@
+<template> 
+    <Layout />
+
+    hello test
```
```suggestion

```

masterskynetx10 commented on resources/js/Pages/Home.vue:
```diff
@@ -0,0 +1,14 @@
+<template>
+    <Layout>
+    <p class="text-black-15 text-xl bg-primary sm:bg-black-15">Heading text</p>
+    </Layout>
```
does your IDE support code format? make sure it automatically does the format. Please remove this sample text and create a new prop for inject
```suggestion
    <Layout>
        <p class="text-black-15 text-xl bg-primary sm:bg-black-15">Heading text</p>
    </Layout>
```
